test/index.js

```diff
@@ -49,6 +50,28 @@ describe( "jQuery CSS Builder", function() {
 
 	});
 
+	describe( "Basic - nested+appdir", function() {
+		var css,
+			files = {
+				"fixtures/version.js": fs.readFileSync( __dirname + "/fixtures/nested/version.js" ),
+				"fixtures/widgets/input.js": fs.readFileSync( __dirname + "/fixtures/nested/widgets/input.js" ),
+				"fixtures/theme/version.css": fs.readFileSync( __dirname + "/fixtures/nested/theme/version.css" ),
+				"fixtures/theme/input.css": fs.readFileSync( __dirname + "/fixtures/nested/theme/input.css" )
+			};
+
+		before(function( done ) {
+			jQueryCSSBuilder( files, "structure", { appDir: "fixtures", include: [ "widgets/input" ] }, function( error, _css ) {
+				css = _css;
+				done( error );
+			});
+		});
+
+		it( "should build just fine", function() {
+			expect( css ).to.equal( "input {}\nbody {}\n" );
```

Comment:
This passes as-is, but I don't think it should. Since `input.js` depends on `version.js`, their respective CSS should reflect that dependency, so this expect should be equal to `"body {}\ninput {}\n"`

@arschmitz @rxaviers thoughts?